aws/src/test/java/org/apache/iceberg/aws/AwsClientPropertiesTest.java

```diff
@@ -140,4 +141,54 @@ public void refreshCredentialsEndpointSetButRefreshDisabled() {
     assertThat(awsClientProperties.credentialsProvider("key", "secret", "token"))
         .isInstanceOf(StaticCredentialsProvider.class);
   }
+
+  @Test
```

Comment:
Should we add a test here that resolves the relative path?

Reply:
done